src/Spec2-Commander2/SpInvalidUserInputError.extension.st

```diff
@@ -3,7 +3,7 @@ Extension { #name : 'SpInvalidUserInputError' }
 { #category : '*Spec2-Commander2' }
 SpInvalidUserInputError >> actForSpec [
 	"Does nothing on purpose."
-	UIManager default
-		inform: self messageText.
+
+	SpApplication defaultApplication inform: self messageText.
```

Comment:
Don't you have a reference to the SpApplication instance?
Because you are not sure the application that triggered `SpInvalidUserInputError` is the default or not. It is better to avoid the use of globals.
If you don't have it, you could pass it to SpInvalidUserInputError when creating the error.